rs/sns/testing/tests/sns_testing_ci.rs

```diff
@@ -72,20 +56,38 @@ async fn prepare_network_for_test(
 ) -> (PocketIc, NeuronId) {
     // Preparing the PocketIC-based network
 
+    let all_icp_features = IcpFeatures {
+        registry: Some(EmptyConfig {}),
```

Comment:
How come each feature has a config of the same type?

Reply:
Sounds good. Just note that once you decide to have any non-trivial config, assuming it's not going to be the same for each feature, then either you would need to store those configs dynamically (similar to, e.g., JSON) or you would need to specialize the argument types (e.g., `EmptyRegistryConfig`).